.gitignore

```diff
@@ -1,3 +1,6 @@
 .project
 .idea
 node_modules
+!intl/en/
+intl/*
+
```

Comment:
I m not sure who originally decided the three line's standard, but if the original decider requires three, then probably we should keep `!intl/`.
ATM i think the only difference is if a repo doesn't have folder `en`, `!intl` makes sure the `intl` folder(empty in this case) still exist in that repo